daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/computedLengthFields.tdml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<testSuite suiteName="Computing Length Fields"
+           xmlns="http://www.ibm.com/xmlns/dfdl/testData"
+           xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:ex="http://example.com">
+
+  <tdml:defineSchema name="s" useDefaultNamespace="false" elementFormDefault="unqualified"
```

Comment:
These files look weird without the tdml prefix, but I think I just need to get used to it. You did miss this one spot though. Should we completely remove the xmlns:tdml prefix namespace mapping if the goal is to not need them?

Reply:
No you can't drop this one.

This style is what I now prefer. Much less prefix clutter. 

You need the tdml prefix on this element because it is redefiing the default prefix to bhe XML Schema's for the body of the define. The element itself would not be in the right namespace if you removed the tdml: prefix.

So you can't really drop the prefix definitions. You still need them sometimes.

Reply:
Ah right, I didn't notice you redefined xmlns, makes sense. :+1: